parsely/src/main/java/com/parsely/parselyandroid/ParselyTracker.java

```diff
@@ -428,7 +427,7 @@ private void enqueueEvent(Map<String, Object> event) {
         // Push it onto the queue
```

Comment:
Suggestion (💡): Consider replacing `storageSizeLimit` with `STORAGE_SIZE_LIMIT` for the comment on the `enqueueEvent(...)` method.